transfer.md

```diff
@@ -93,14 +93,12 @@ Those Docker commands currently do not have equivalents in `podman`:
 | :--- | :--- |
 | `docker container update`  | podman does not support altering running containers. We recommend recreating containers with the correct arguments.|
 | `docker container rename`   | podman does not support `container rename` - or the `rename` shorthand. We recommend using `podman rm` and  `podman create` to create a container with a specific name.|
-| `docker network`  ||
 | `docker node`     ||
 | `docker plugin`   | podman does not support plugins.  We recommend you use alternative OCI Runtimes or OCI Runtime Hooks to alter behavior of podman.|
 | `docker secret`   ||
 | `docker service`  ||
 | `docker stack`    ||
 | `docker swarm`    | podman does not support swarm.  We support Kubernetes for orchestration using [CRI-O](https://github.com/cri-o/cri-o).|
-| `docker volume`   | podman currently supports file volumes.  Future enhancement planned to support Docker Volumes Plugins
```

Comment:
Shouldn't this line and line 96 be moved into the "Development Transfer" section rather than just removed?

Reply:
We already included all volume commands. I added the network commands.